labrad/protocol.py

```diff
@@ -333,5 +373,92 @@ def __init__(self, source, context, target):
     def __repr__(self):
         return 'MessageContext(source=%s, ID=%s, target=%s)' % (self.source, self.ID, self.target)
 
-# factory for churning out LabRAD connections
+
+# factory for creating LabRAD connections
 factory = protocol.ClientCreator(reactor, LabradProtocol)
```

Comment:
It was a little puzzling to me because I'm not used to anything other than functions, classes, modules, and constants sitting at the module level. I know there's not a law against having other objects in the module, it's just rare enough that it was puzzling.